tools/expotools/src/commands/ClientBuild.ts

```diff
@@ -100,7 +92,8 @@ async function askToRecreateSimulatorBuildAsync(): Promise<boolean> {
 
 async function askToOverrideBuildAsync(): Promise<boolean> {
   if (process.env.CI) {
-    return true;
+    // we should never override anything in CI, too easy to accidentally mess something up in prod
+    return false;
```

Comment:
This is probably not correct — imho we do want to override builds on the CI. If we're during the release process (and let's say version `2.17.0` hasn't been released to production yet), we might want to override the simulator build (only on staging) without changing the version number (which would require additional manual work).